config/commitlint.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: [
+    '@commitlint/config-conventional',
+  ],
+  rules: {
+    'footer-leading-blank': [2, 'always'],
+    'body-leading-blank': [2, 'always'],
+    'subject-exclamation-mark': [2, 'never'],
```

Comment:
I realise the first few commits have `!` but the rest don't, even though it is breaking change. Moving forward, are we not going to use `!` because from the commit itself is hard to tell if it is a breaking change unless we click in?

Reply:
The issue what the `!` was not recognised by semantic-release completely ignoring it. I think it was because semantic-release was using `angular` preset rather than `conventionalcommits` I think we can restore it back 🤔

Reply:
Hmmm, will this affect your commits since some of them have `!` and some don't? I'm not sure about this.

Reply:
No, in conventionalcommits breaking changes are identified by `!` or `BREAKING CHANGE`

Reply:
Ah okay 👌